lib/mobile/device/appetize-application-manager.ts

```diff
@@ -0,0 +1,58 @@
+import { EventEmitter } from "events";
+
+export class AppetizeApplicationManager extends EventEmitter implements Mobile.IDeviceApplicationManager {
```

Comment:
We actually do because mobile-cli's [event-emitter](https://github.com/telerik/mobile-cli-lib/blob/master/appbuilder/device-emitter.ts#L42) attaches events to the device's application manager.
You can treat this as reserved for future use 😄